src/schemas/json/sublime-syntax.json

```diff
@@ -0,0 +1,248 @@
+{
+  "$comment": "https://www.sublimetext.com/docs/syntax.html",
+  "$id": "https://json.schemastore.org/sublime-syntax.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "definitions": {
+    "meta_pattern": {
+      "type": "object",
```

Comment:
Add `title` for `object` keys as they are shown via YAML extension. Also, it's beneficial to add titles for everything. It's especially useful for objects.